client/app/components/ownership-service-modal/ownership-service-modal.html

```diff
@@ -0,0 +1,39 @@
+<div class="modal-header">
+  <button type="button" class="close" ng-click="$dismiss()" aria-hidden="true">
+    <span class="pficon pficon-close"></span>
+  </button>
+  <h4 class="modal-title" id="myModalLabel" translate>Set Service Ownership</h4>
+</div>
+<div class="modal-body">
+  <form class="form-horizontal">
+    <div pf-form-group pf-input-class="col-sm-8" pf-label-class="col-sm-3" pf-label="{{'Select an Owner'|translate}}">
+      <select ng-model="vm.modalData.resource.owner.userid" class="form-control">
+        <option ng-repeat="user in vm.users.resources | orderBy:'name'"
```

Comment:
How long will the array usually be? Written like this, it will try to sort it on every digest, so maybe we might want to do this just once and repeat over an already sorted array?